EKS-POD-IDENTITY-README.md

```diff
@@ -0,0 +1,160 @@
+centralized hub for building, deploying, and monitoring models. 
+
+In a previous [article] (http://ADD_LINK_HERE) we saw how IAM Role for Service Accounts (IRSA) can be utilized within 
+Domino Workloads for assuming IAM Roles. There are situations where IRSA can be operationally challenging to use.
+AWS provides another mechanism to assume IAM roles within Kubernetes workloads known as EKS Pod Identity to address
+these challenges.
+
+In this article we will explore what these operational challenges are and how EKS Pod Identity can be used within 
+Domino Workloads. We will look at specific situations where this approach is more suitable than the IRSA approach.
+
+While this document is meant to guide you on configuring AWS IAM Role propagation to Domino Workloads based on 
+EKS Pod Identity, please speak with your Domino Professional 
+Services contact for additional information, assistance and support.
+
+
+### Operational Challenges of IRSA
+
+In general if you treat the EKS cluster hosting Domino as a "pet" and not "cattle", IRSA is a better choice. It offers
+more flexibility with respect to assigning roles to Domino workloads. It can also be argued that, delegating the mapping
+of Kubernetes Service Accounts to IAM Role via a separate IAM team is a good security practice which allows for more
+safeguards when assigning roles to Kubernetes workloads.
+
+However, if you use any variant of Blue/Green deployments, where an upgrade to Domino is a brand new EKS cluster,
+you will find EKS Pod Identities useful.
+
+In general the EKS administrators or Domino administrators, do not own the AWS Identity and Access Management. For any
+AWS role which can be assumed by a Domino workloads, the IAM owner has to update the trust policy of the IAM Role to
+explicity point to the OIDC provider associated with the EKS cluster. This OIDC provider is unique for each EKS cluster.
+
+Consequently, in a Blue/Green scenario when you start the Green EKS cluster and install Domino on it, you will need
+to modify the trust relationship of each IAM Role assumed by the Domino Workload to include the OIDC provider of the
+Green Domino. After the Blue EKS cluster is retired, you will need to remove the references to the OIDC provider of the
+Blue EKS cluster. This increases the operational burden on the IAM Team. In most cases, your internal policies won't 
+permit you to make these changes without proper approvals in place adding more pressures on the operational process. 
+
+EKS Pod Identities are introduced by AWS to address this operational challenge
+
+### EKS Pod Identities
+
+
+What makes EKS Pod Identities impervious to the challenge of changing EKS clusters?  The key is understanding who
+the underlying Principal is in the trust policy.
+
+In IRSA your trust policy for the IAM Role looks something like this
+```json
+{
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "$PROVIDER_ARN"
+      },
```

Comment:
We should add in an actual endpoint template here just to make it clearer, instead of using variables

Reply:
Done